rfcs/0097-no-read-store-dir.md

```diff
@@ -0,0 +1,78 @@
+---
+feature: nix-store-perms
+start-date: 2021-07-04
+author: Las Safin
+co-authors:
+shepherd-team: @kevincox @7c6f434c @edolstra
+shepherd-leader: @edolstra
+related-issues:
+---
+
+# Summary
+[summary]: #summary
+
+- NixOS should have a module for configuring the permissions set for `/nix/store` on boot.
+- Nix should not enforce the permissions used for `/nix/store`.
+- The default permissions if the store doesn't exist should be 1735 when the store is made by Nix or the NixOS installer.
+  This means that the nixbld group can't `ls` the directory.
+
+# Motivation
+[motivation]: #motivation
+
+Right now you can't set the permissions for `/nix/store`, since they'll be overwritten
+by Nix anytime you use `nix`.
+
+We want to `chmod g-r /nix/store`, because the `nixbld` group doesn't actually
+need to read the directory. It only needs to be able to write and "execute" it.
+This, however, should be optional, since the user should be able to do what they want.
```

Comment:
```suggestion
This, however, should be optional, since the user should be able to configure the permissions however they want.
```

Nit: define the subject more clearly.